src/Components/Book.module.css

```diff
@@ -1,9 +1,101 @@
+.article {
+  display: flex;
+  justify-content: space-between;
+  background-color: white;
+  padding: 2rem;
+}
+
 .book_btn_container {
   display: flex;
```

Comment:
- [ ] Let us kindly avoid repeating the same CSS rule, for example, `display: flex, flex-direction: column` has appeared a few times, we can create a class for it and apply it on the HTML element